Docs/Practitioners/PdcPractitioner.md

```diff
@@ -0,0 +1,28 @@
+# PdcPractitioner
+
+Holds the board order and licensing information of the medical professional
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| fid  | string | Federation ID of the medical professional |
+| boardOrders | [BoardOrder](#boardOrder) array| A collection of the professional's board orders |
+| licenses | [License](#license) array | a collection of the professional's licenses |
+
+## Board Order
```

Comment:
This type you separated out the names. Be consistent. In JSON there is no type "name" so we tend to just keep the name together even though it is harder to read. This is an API after all.